waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim

```diff
@@ -945,20 +976,25 @@ proc subscribeToGroupEvents(ethClientUri: string, ethAccountAddress: Option[Addr
 
   proc startSubscription(web3: Web3) {.async, gcsafe.} =
```

Comment:
Why do we need to have a separate proc for this, why not just call the `subscribeToMemberRegistrations` proc?

Reply:
Eventually, when we have the reconnection logic, it becomes easier to resubscribe to events from the last processed block.